### ISSUE TITLE
Questions about the experiments.

### ISSUE THREAD

**Issue by mingyuShin** [valeoai/MFEval#2]
Thank you for your great work!

1. The reason why the numbers for ViP3D and UniAD in Table 1 differ from those in the papers of ViP3D and UniAD.
2. In the case of UniAD, whether the learning and finetuning included training the occformer and planner as well.
3. When measuring the motion forecasting metric, how was the average calculated for categories, and whether it included averaging together vehicles (car, truck, construction vehicle, bus, trailer, motorcycle, and bicycle) or pedestrians as well.

**Comment by yihongXU** [comment]
Hi,

Thank you for your interest in our work.
1) It is because in ViP3D and UniAD, pedestrians are also included in their evaluation, also many of them are static objects, while in NuScenes Prediction and our benchmarks, those objects can bias the study and are not included.
2) For the map study, we follow the second stage training of UniAD.
3) We only average the performance of  ['bus', 'car', 'truck']. 

best,

**Comment by mingyuShin** [comment]
Thank you for your fast reply!
